clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -126,6 +126,11 @@ New checks
   reference. This may cause use-after-free errors if the caller uses xvalues as
   arguments.
 
+- New :doc:`bugprone-pointer-arithmetic-on-polymorphic-object
+  <clang-tidy/checks/bugprone/pointer-arithmetic-on-polymorphic-object>` check.
+
+  Finds pointer arithmetic performed on classes that declare a virtual function.
```

Comment:
Change this description to point about polymorphic-object. Or instead of that declare (class does not declare anything), just say "that contain".